### ISSUE TITLE
[DOCUMENTATION] Mean Annual soil Erosion rate

### ISSUE THREAD

**Issue by SethCallewaert** [watem-sedem/watem-sedem#56]
Documentation references: https://github.com/cn-ws/cn-ws/blame/master/docs/watem-sedem.rst#L73 (and on the whole page)

Please use either use option A) or B) and fill in the selected options:

**B. I have a question considering this section:**

In the text there is often referred to the mean annual soil erosion rate, however I feel like RUSLE give a total/potential/maximal/calculated erosion rate per pixel. So it is not clear to me if here the pixel values are referred to or the mean value of all the calculated pixels. 

I would rather choose a less ambiguous name for the annual erosion rate (I feel like within our other studies we mostly use potential soil erosion). 

To nitpick even harder I would like to propose the letter A for the annual soil erosion instead of E (conform to most literature), and then E could for example be used for Net erosion (see https://github.com/cn-ws/cn-ws/issues/55)

**Comment by Sachagobeyn** [comment]
> In the text there is often referred to the mean annual soil erosion rate, however I feel like RUSLE give a total/potential/maximal/calculated erosion rate per pixel. So it is not clear to me if here the pixel values are referred to or the mean value of all the calculated pixels.

Good point. I'm not sure if we can translate RUSLE to ``total/potential/maximal/calculated erosion rate per pixel`` as RUSLE is a theoretical concept, outside the raster/pixel context of CN-WS. Looking at Renard et al. (1997):

![afbeelding](https://github.com/cn-ws/cn-ws/assets/9808350/1819a8e9-bfb4-4282-93f0-a5b31f1ee7b9)

and

![afbeelding](https://github.com/cn-ws/cn-ws/assets/9808350/66ef7a23-83e9-4ad5-b16d-2ac92854bd8b)

we should define RUSLE as ``A`` and as ``mean annual soil erosion rate``. ``E`` should be defined as:

![afbeelding](https://github.com/cn-ws/cn-ws/assets/9808350/bc8890d3-d2ff-4558-be52-1c1a6b3a374d)

Erosion, or ``net erosion`` as you stated. 

In summary, @SethCallewaert , ok?:

- A: mean annual soil erosion rate- theoretical gross erosion concept.
- E: netto erosion - theoretical net erosion concept.
- Symbols: A and E are not mentioned/referred in case we talk about model applications (raster/pixel)
